### PR TITLE
Added additional data items to Income and Allowance and deductions se…

### PR DESCRIPTION
**Pull request by chandraAndem**
…ctions

### CODE REVIEW

**Review by MihailsC-hmrc — APPROVED** [review]
LGTM.
Some notes are left at your discretion.

File: conf/messages [hunk 1]
```diff
@@ -189,6 +200,10 @@ deduction_breakdown.table.personal_allowance                           = Persona
 deduction_breakdown.table.pension_contributions                        = Pensions contributions
 deduction_breakdown.table.loss_relief                                  = Loss relief
 deduction_breakdown.table.gift_of_investments_and_property_to_charity  = Gift of investments and property to charity
+deduction_breakdown.table.annual_payments                              = Annual Payments
+deduction_breakdown.table.qualifying_loan_interest                     = Qualifing loan interest
```
[the remark this answers is not in the document]
Comment (chandraAndem) [reply conf/messages]: Changed

File: app/models/calculation/Calculation.scala [hunk 2]
```diff
@@ -36,7 +36,15 @@ case class PayPensionsProfit(totalSelfEmploymentProfit: Option[BigDecimal] = Non
...
-														 totalStateBenefitsIncome: Option[BigDecimal] = None
+														 totalStateBenefitsIncome: Option[BigDecimal] = None,
+														 totalFHLPropertyProfit: Option[BigDecimal] = None,
+														 totalForeignPropertyProfit: Option[BigDecimal] = None,
+														 totalEeaFhlProfit: Option[BigDecimal] = None,
+														 totalOverseasPensionsStateBenefitsRoyalties: Option[BigDecimal] = None,
+														 totalAllOtherIncomeReceivedWhilstAbroad: Option[BigDecimal] = None,
+														 totalOverseasIncomeAndGains: Option[BigDecimal] = None,
+														 totalForeignBenefitsAndGifts: Option[BigDecimal] = None,
+														 totalShareSchemesIncome: Option[BigDecimal] = None
```
[the remark this answers is not in the document]
Comment (MihailsC-hmrc) [reply app/models/calculation/Calculation.scala:47]: Looks like we'll fix the alignment i.e. tab characters in some next tech debt pr.

**Review by MihailsC-hmrc — APPROVED** [review]
LGTM

**Pull request merged**